windows-lb/variables.tf

```diff
@@ -64,20 +75,38 @@ variable "image_os_version" {
   default = "Server 2012 R2 Standard"
 }
 
-variable "subnet_cidr" {
-  type    = list(string)
-  default = ["10.1.20.0/24", "10.1.21.0/24"]
+variable "vcn_cidr" {
+  default = "10.1.0.0/16"
 }
 
+variable "subnet1_cidr" {
+  default = "10.1.20.0/24"
+}
+
+variable "subnet2_cidr" {
+  default = "10.1.21.0/24"
+}
+
+variable "mssql_primary_shape" {
+  default = "VM.Standard2.2"
+}
+
+variable "mssql_standby_shape" {
+  default = "VM.Standard2.2"
+}
+
+# Dictionary Locals
 locals {
   compute_flexible_shapes = [
     "VM.Standard.E3.Flex",
-    "VM.Standard.E4.Flex"
-  ]
-  compute_shape_flexible_descriptions = [
-    "Cores for Standard.E3.Flex and BM.Standard.E3.128 Instances",
-    "Cores for Standard.E4.Flex and BM.Standard.E4.128 Instances"
+    "VM.Standard.E4.Flex",
+    "VM.Optimized3.Flex",
+    "VM.Standard.A1.Flex"
```

Comment:
A1 shapes will not support Windows images. This should be unsupported

Reply:
@jeevanjoseph ok, I will remove it.